papers/templates/papers/search.html

```diff
@@ -201,7 +202,13 @@
 
 <span class="detailsTitle">{% trans "Search" %}</span>
 <div class="detailsContent">
-<p id="nbPapersFound">{{ nb_results }}</p>
+    <p id="nbPapersFound">
+        {% blocktrans trimmed with formattedCounter=nb_results|intcomma count counter=nb_results|default:0 %}
+            {{ formattedCounter }} paper found
+            {% plural %}
+            {{ formattedCounter }} papers found
+        {% endblocktrans %}
+    </p>
```

Comment:
It makes sense to do this on the frontend! And the translation system is much better like that. However, this causes issues when dynamically updating the number of results in Javascript: the " papers found" text is not appended to the number. I think we need to use the JS localization system here.